src/lib/rest/rest.cpp

```diff
@@ -666,9 +667,9 @@ static int contentTypeCheck(ConnectionInfo* ciP)
   // Five cases:
   //   1. If there is no payload, the Content-Type is not interesting
   //   2. Payload present but no Content-Type 
-  //   3. text/xml used and acceptTextXml is setto true (iotAgent only)
+  //   3. text/xml used and acceptTextXml is set to true (iotAgent only)
```

Comment:
Off-topic: the existence of "iotAgent" only make me think that we have deprecated code related with old iotAgent (which as far as I know it no longer under developed). That code should be removed, in the same way ConfMan code was recently removed.

Could you add a tech debt issue about it, pls? I think that "tech debt should" would be fine.